keras/utils/data_utils.py

```diff
@@ -434,15 +434,15 @@ class OrderedEnqueuer(SequenceEnqueuer):
     # Arguments
         sequence: A `keras.utils.data_utils.Sequence` object.
         use_multiprocessing: use multiprocessing if True, otherwise threading
-        scheduling: Sequential querying of datas if 'sequential', random otherwise.
+        ordered: Sequential querying of datas if `True`, random otherwise.
     """
 
     def __init__(self, sequence,
                  use_multiprocessing=False,
-                 scheduling='sequential'):
+                 ordered=True):
         self.sequence = sequence
         self.use_multiprocessing = use_multiprocessing
-        self.scheduling = scheduling
+        self.ordered = ordered
```

Comment:
Just a test that would validate that if ordered=False, there is some shuffling. Nothing fancy, but I could prevent some bugs in the future.